src/routes/Accounts/AccountTabs/Keys.tsx

```diff
@@ -84,26 +84,28 @@ const AccountKeys: FC<AccountKeysProps> = ({ account, exportAccount }) => {
   return (
     <Flex mb="4rem">
       <Box w="37.25rem">
```

Comment:
If phrase is null, means the account has no spend key and we can exclude this box as an export mechanism